pubpub/pubpub
# feature: Show the current Page in the Dashboard dropdown

Resolves #1062  (well, sorta)

One of the most common complaints about PubPub is that there's no easy way to navigate back to the Dashboard for a Page while viewing the Page. This PR changes that by adding such a link to the `ScopeDropdown` (Dashboard) menu:

![image](https://user-images.githubusercontent.com/2208769/96747872-d4bb5100-1396-11eb-8057-923d848ca668.png)

_Test plan:_
- Visit a Page and ensure that it appears in the Dashboard menu
- Check that the Page does not appear in the `ScopeDropdown` in the left-hand sidebar of the Dashboard itself
